.pre-commit-config.yaml

```diff
@@ -1,3 +1,6 @@
+default_language_version:
```

Comment:
I don't think this should be necessary at all -- certainly, CI passes with the defaults. The only way I can see this being an issue is if you have `python3.7` as your default interpreter but `python3.8` is available in your environment. Is that possible/common outside of say, an old RHEL that we don't really support as a development environment?

Reply:
I encountered this on my Ubuntu 20.04.

Reply:
[Ubuntu 20.04 ships with](https://wiki.ubuntu.com/FocalFossa/ReleaseNotes) Python 3.8 as the default -- according to both the release notes and a fresh VM. I'm going to chalk what you ran into as local configuration strangeness (probably some `update-alternatives` in the background), barring a reproduction in a clean VM/container of a currently supported Linux distro.

Reply:
I'm away from my machine so I'll just remove this and open an issue if this persists.